ecommerce/frontend/src/components/checkout/addresslist/AddressCard.js

```diff
@@ -0,0 +1,76 @@
+import './AddressCard.less'
+import cls from 'classnames'
+import { DeleteOutlined, EditOutlined } from '@ant-design/icons'
+import { useState } from 'react'
+import { AddressModal } from './AddressModal'
+import { Popconfirm, Spin, notification } from 'antd'
+import { sleep } from '../../../utils'
+
+export const AddressCard = ({ address, selected = false, onSelect = () => {}, onAddressInfoChange = () => {} }) => {
+    const [editVisible, setEditVisible] = useState(false)
```

Comment:
making the `div `wrapped with` onSelect` is a super nice user experience, well done.

Reply:
Also, clicking `delete `or `edit `icons select the item. This is bad :)

Reply:
I noticed that it selects on clicking edit&delete. I can see it's not the best behavior but I also think that it's somehow harmless. Can you see a harmful outcome out of this?

Reply:
easy way is to call event.stopPropagation() on child onClick
https://stackoverflow.com/questions/37568550/react-prevent-event-trigger-on-parent-from-child